playbooks/ansible/roles/sit.gpfs/tasks/main.yml

```diff
@@ -75,8 +75,54 @@
     - name: Define block size and mountpoint
       command: "{{ gpfs_toolkit }} filesystem modify -B 128K -m /gpfs/sit_fs sit_fs"
 
+    - name: Configure SMB protocol
+      when: config.be.variant == 'scale'
+      block:
+        - name: Define protocol nodes
+          command: "{{ gpfs_toolkit }} node add {{ item }} -p"
+          loop: "{{ config.groups['cluster'] }}"
+
+        - name: Enable SMB protocol
+          command: "{{ gpfs_toolkit }} enable smb"
+
+        - name: Configure shared directory
+          command: >-
+            {{ gpfs_toolkit }} config protocols -f sit_fs -m /gpfs/sit_fs
+              -e {{
+                    config.nodes |
+                    dict2items |
+                    selectattr('value.ctdb.public', 'defined') |
+                    map(attribute='value.ctdb.public') |
+                    join(',')
+              }}
```

Comment:
What are we configuring here?

Reply:
This is the set of floating IPs that will be dynamically assigned to the nodes serving SMB protocol. Equivalent to what we normally use for ctdb. That's why I use the same pool of IPs.